posttest.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var findTables = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'T\'' + ';';
+var findViews = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'V\'' + ';';
+
+if (!process.env.CI)
+  return console.log('Skipping post-test ...');
+console.log('> Cleaning up the database ...');
+
+// drop all the tables and views from the schema
```

Comment:
I mean 
```js

async.series([
  function dropViews(cb) {
    db.connector.execute('find_view', function(err, views) {
        async.each(views, dropView, cb); 
    })
  },
  function dropTables(cb) {
    db.connector.execute('find_table', function(err, tables) {
        async.each(tables, dropTable, cb); 
    })
  }
], function(err) {
  if (err) throw err;
})
```

By the current code, which equals to 
```js
async.each(collection, processItem, function(err, result) {
    asyncFunction();
})
```
My concern is the whole `async.each` function will end before you get the returned result of `asyncFunction()`